Made change for MySQL usage that was missing

mathieupotier opened the pull request:
Hi I made few change in order to have the mysql dependencies satisfied and working.

mathieupotier committed:
add pip required module for mysql

mathieupotier committed:
fix the build

mathieupotier committed:
fix dockerfile

mgodlewski commented:
Hi

Thanks for your work ! I wasn't expected any pull request :)
Could you squash your commits and add in the README file that mysql is also supported ?

Regards

mathieupotier committed:
Update readme

Add Table for all env var and show example for SQLite run, MySQL run or PostgreSQL.

mathieupotier commented:
Done.

If I remember well you can choose to "Squash & Merge" the PR, if not you can active this behaviour on the repository settings.

Regards

PS: I'm preparing another PR soon to fix behaviour on init with DB: when creating a blank DB, the schema is not up to date for some reason, it requires to run a refreshall command. Time for me to test it well.

mgodlewski commented:
Nice !
Thanks for your work, the updated image is building : https://hub.docker.com/r/mgodlewski/diacamma/builds/bepff7e6aq6g6gcpgkdls6h/
